### ISSUE TITLE
OC8: Updating contacts fails 

### ISSUE THREAD

**Issue by derkostka** [owncloud/contacts#767]
Version: Owncloud 8 alpha 1

Reproduce: Change contacts thumbnail on a mobile device (android with sync via davdroid)

Error: Operation fails, error is attached (in operation cacheThumbnail)

Log: 

"app":"PHP","message":"Undefined index: remove at \/var\/www\/owncloud\/apps\/contacts\/lib\/utils\/properties.php#303","level":3,"time":"January 04, 2015 17:56:40"}

**Comment by derkostka** [comment]
@LEDfan : The index "remove" is not inside the array, maybe this is just a bug in the order of operations.

**Comment by EtienneM** [comment]
I have the same issue here.

**Comment by enoch85** [comment]
Just encounterd the same issue. 

Triggerd by removing a number in a Contact.

```
Undefined index: remove at /var/www/owncloud/apps/contacts/lib/utils/properties.php#303 
```
Ping @LEDfan

**Comment by derkostka** [comment]
@LEDfan and still on 8.0.3 RC1. Can you please prioritize this one ? Maxbe this is related to the xCache ?
"app":"PHP","message":"Undefined index: remove at \/var\/www\/owncloud\/apps2\/contacts\/lib\/utils\/properties.php#303","level":3,"time":"April 11, 2015 10:19:51"}

**Comment by derkostka** [comment]
fixed by #877 , please consider backpoirting to OC 7 and OC 8.0.3. Thank you !

**Comment by LEDfan** [comment]
I'll have a look at this today.
